source/elements/oneMKL/source/architecture/other_architecture.inc.rst

```diff
@@ -11,10 +11,41 @@ This section covers all other features in the design of oneMKL architecture.
 
 .. _onemkl_spec_current_version:
 
-Current Version of this oneMKL Specification
+Specification Version and Compliance
 +++++++++++++++++++++++++++++++++++++++++++++
 
-This is the oneMKL specification which is part of the oneAPI specification version 1.0.0.
+Each oneMKL domain must define a preprocessor macro to represent the version of the specification that the implementation is compliant with.
+
+The macros for each domain are listed as follows:
+
+  | ONEMKL_BLAS_SPEC_VERSION
+  | ONEMKL_LAPACK_SPEC_VERSION
+  | ONEMKL_SPBLAS_SPEC_VERSION
+  | ONEMKL_DFT_SPEC_VERSION
+  | ONEMKL_RNG_SPEC_VERSION
+  | ONEMKL_STAT_SPEC_VERSION
+  | ONEMKL_VM_SPEC_VERSION
+
+The specification version can be created by appending all digits of the specification version in the format of <MAJOR><MINOR>. MINOR version always uses two digits. This version can be used to check the compatibility of the implementation with the specification version. Note that the revision is not included here because it reflects changes only for the specification document without affecting the implementation. If the implementation is not compliant with any release of the specification, then the macro must have a numerical value of `000`.
```

Comment:
Should we link to a document that describes how/when the MAJOR, MINOR, and revision versions get updated?

Reply:
Added at the end of this section.